src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -366,26 +366,70 @@ function showNewlyAddedResponseCommentEditForm(addedIndex) {
     $("#responseCommentEditForm-" + addedIndex).show();
 }
 
-function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, sender) {
+function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, clickedElement) {
     $(".tooltip").hide();
-    var panelBody = $(sender).parent().find('div[class^="panel-body"]');
+    var $clickedElement = $(clickedElement);
+    var panelBody = $clickedElement.parent().find('div[class^="panel-body"]');
     var fsNameForUrl = encodeURIComponent(fsName);
     var url = "/page/instructorFeedbackResponseCommentsLoad?user=" + user + "&courseid=" + courseId + "&fsname=" + fsNameForUrl + "&fsindex=" + fsIndx;
-    $(sender).find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
+    
+    // If the content is already loaded, toggle the chevron and exit.
+    if ($clickedElement.hasClass("loaded")) {
+        $clickedElement.siblings(".collapse").clearQueue().collapse("toggle");
```

Comment:
Since this line of code is repeated and is slightly lengthy, we can move this it into a function so that the function name can explain what the code does.